.markdownlintignore

```diff
@@ -0,0 +1,12 @@
+api
+catalog/node_modules
```

Comment:
might make sense together to put it in the group with `.git` and `venv`

Reply:
Agree